sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/lro/AnalyzeExtractiveSummarization.java

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.lro;
+
+import com.azure.ai.textanalytics.TextAnalyticsClient;
+import com.azure.ai.textanalytics.TextAnalyticsClientBuilder;
+import com.azure.ai.textanalytics.models.AnalyzeActionsOperationDetail;
+import com.azure.ai.textanalytics.models.AnalyzeActionsOptions;
+import com.azure.ai.textanalytics.models.AnalyzeActionsResult;
+import com.azure.ai.textanalytics.models.ExtractSummaryAction;
+import com.azure.ai.textanalytics.models.ExtractSummaryActionResult;
+import com.azure.ai.textanalytics.models.ExtractSummaryResult;
+import com.azure.ai.textanalytics.models.SummarySentence;
+import com.azure.ai.textanalytics.models.SummarySentencesOrder;
+import com.azure.ai.textanalytics.models.TextAnalyticsActions;
+import com.azure.ai.textanalytics.models.TextDocumentInput;
+import com.azure.ai.textanalytics.util.AnalyzeActionsResultPagedIterable;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.http.rest.PagedResponse;
+import com.azure.core.util.Context;
+import com.azure.core.util.polling.SyncPoller;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AnalyzeExtractiveSummarization {
+    /**
+     * Main method to invoke this demo about how to analyze a batch of tasks.
+     *
+     * @param args Unused arguments to the program.
+     */
+    public static void main(String[] args) {
+        TextAnalyticsClient client = new TextAnalyticsClientBuilder()
+                                         .credential(new AzureKeyCredential("{key}"))
+                                         .endpoint("{endpoint}")
+                                         .buildClient();
+
+        List<TextDocumentInput> documents = new ArrayList<>();
+        for (int i = 0; i < 21; i++) {
+            documents.add(new TextDocumentInput(Integer.toString(i),
+                "The government of British Prime Minster Theresa May has been plunged into turmoil with "
```

Comment:
should we call out where this document was taken from?

Reply:
It is from the service's documentation. I directly copy and paste the context from the Extractive Summarization example that service shared.